ArchiSteamFarm/IPC/ArchiKestrel.cs

```diff
@@ -60,9 +62,14 @@ internal static class ArchiKestrel {
 			// The order of dependency injection matters, pay attention to it
 			IWebHostBuilder builder = new WebHostBuilder();
 
+			string customDirectory = Path.Combine(Directory.GetCurrentDirectory(), SharedInfo.WebsiteDirectory);
```

Comment:
Now add comment describing why absolute path is needed and test if everything works, including your `GET /Api/WWW/Directory` that you modified in other file.